python/packages/devui/pyproject.toml

```diff
@@ -4,9 +4,9 @@ description = "Debug UI for Microsoft Agent Framework with OpenAI-compatible API
 authors = [{ name = "Microsoft", email = "af-support@microsoft.com"}]
 readme = "README.md"
 requires-python = ">=3.10"
-version = "1.0.0b251001"
+version = "1.0.0-b251001"
 license-files = ["LICENSE"]
-urls.homepage = "https://aka.ms/agent-framework"
+urls.homepage = "https://github.com/microsoft/agent-framework"
```

Comment:
The git repo is already linked as the Source, no need for that twice, so leave this to point to docs